features/i8n.feature

```diff
@@ -0,0 +1,118 @@
+Feature: Pro supports multiple languages
+
+    @series.all
+    @uses.config.machine_type.lxd.container
+    @uses.config.contract_token
+    Scenario Outline: Pro client's commands run successfully
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        ## Change the locale
+        When I run `locale-gen uk_UA.UTF-8` with sudo
+        And I run `update-locale LANG=uk_UA.UTF-8` with sudo
```

Comment:
I'm not familiar with changing locales but I was expecting to see the installation of a language-pack somewhere like `apt install language-pack-uk`?

I'm also confused about the locale-gen command since the manpage and help text don't indicate that it accepts a positional argument like it's being used here.

But again I'm not familiar with switching locales so IDK :upside_down_face: